cmd/legacy_param_mapper_test.go

```diff
@@ -165,7 +191,13 @@ func TestPopulateConfigFromLegacyFlags(t *testing.T) {
 				},
 				Logging: cfg.LoggingConfig{
 					FilePath: cfg.ResolvedPath("/tmp/log-file.json"),
+					Severity: cfg.LogSeverity(cfg.TRACE), // Because debug fuse flag is set.
```

Comment:
"TRACE" is good enough.
```suggestion
					Severity: "TRACE", // Because debug fuse flag is set.
```